docs/gen_ref_pages.py

```diff
@@ -28,9 +30,7 @@
             continue
         if parts == ():
             continue
-        if parts[0] == "tests":
-            continue
-        if parts[0] == "templates":
+        if parts[0] in exclude_paths:
```

Comment:
Thanks for this haha

Reply:
thansk to you ! :) (you did that change haha)